components/core/src/clp_s/search/AddTimestampConditions.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef CLP_S_ADD_TIMESTAMP_CONDITIONS_HPP
+#define CLP_S_ADD_TIMESTAMP_CONDITIONS_HPP
+
+#include <optional>
+
+#include "../Defs.hpp"
+#include "../TimestampDictionaryReader.hpp"
+#include "Transformation.hpp"
+
+namespace clp_s::search {
+class AddTimestampConditions : public Transformation {
```

Comment:
* Add docstring.
* This name seems more general than we mean. Would `AddAuthoritativeTimestampConditions` be better? On the other hand, I wonder if we really need to restrict it to the authoritative timestamp column. Could we instead figure out the authoritative timestamp column outside of this transformation and then just pass in the tokenized name?